docs/release.md

```diff
@@ -0,0 +1,9 @@
+# Overview 
+
+This document helps stakeholders to agree on release flow.
```

Comment:
@hussein-aitlahcen @seunlanlege are we ok with this?

Reply:
Yes